File: src/OnlineShoppingWeb/cart-ts/cart.model.ts
import { Product } from './Product.model';
import { SaveForLater } from './SaveForLater.model';

export class Cart {
    public id: string;
    public products: Array<Product>;
    public saveForLaters: Array<SaveForLater>;
    public totalPrice: number;
    public productPrice: number;
    constructor() {
    }
}

File: src/OnlineShoppingWeb/ApiModel/CartModel.cs
using OnlineShoppingWeb.Enities;
using System.Collections.Generic;

namespace OnlineShoppingWeb.ApiModel
{
    public class CartModel
    {
        public IEnumerable<Product> Products { get; set; }
        public IEnumerable<SaveForLater> SaveForLaters { get; set; }

        public User User { get; set; }
        public SaveForLater SaveForLater { get; set; }
        public ShoppingCart ShoppingCart { get; set; }
        public decimal totalPrice { get; set; }
        public decimal ProductPrice { get; set; }
    }
}
